From 186195b1a84f2517205522e7ab8e0f62ad61c329 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Reimar=20D=C3=B6ffinger?= <Reimar.Doeffinger@gmx.de>
Date: Tue, 18 Feb 2014 14:48:16 -0800
Subject: [PATCH] Fix incorrect VdpPictureInfo typedef.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

VdpPictureInfo should not be a pointer, otherwise the
VdpDecoderRender function signature asks for a pointer to a pointer.
But it really does expect a pointer directly to e.g. VdpPictureInfoH264.

Signed-off-by: Reimar Döffinger <Reimar.Doeffinger@gmx.de>
Tested-by: Reimar Döffinger <Reimar.Doeffinger@gmx.de>
Reviewed-by: Stephen Warren <swarren@nvidia.com>
Signed-off-by: Aaron Plattner <aplattner@nvidia.com>
---
 include/vdpau/vdpau.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/vdpau/vdpau.h b/include/vdpau/vdpau.h
index 58d5083..6150607 100644
--- a/include/vdpau/vdpau.h
+++ b/include/vdpau/vdpau.h
@@ -2638,7 +2638,7 @@ typedef struct {
 } VdpBitstreamBuffer;
 
 /**
- * \brief A generic "picture information" pointer type.
+ * \brief A generic "picture information" type.
  *
  * This type serves solely to document the expected usage of a
  * generic (void *) function parameter. In actual usage, the
@@ -2647,7 +2647,7 @@ typedef struct {
  * picking the type appropriate for the decoder object in
  * question.
  */
-typedef void * VdpPictureInfo;
+typedef void VdpPictureInfo;
 
 /**
  * \brief Picture parameter information for an MPEG 1 or MPEG 2
-- 
1.9.2

